rename package to `rubicon_ml` (#93)
capitalone/rubicon-ml: docs/source/index.rst

.. _rubicon:

===========
rubicon-ml
===========

Purpose
=======

rubicon-ml is a data science tool that captures and stores model training and
execution information, like parameters and outcomes, in a repeatable and

> searchable way. It's git integration associates these inputs and outputs

Reviewer: ```suggestion
searchable way. Its git integration associates these inputs and outputs
```

or alternatively:

``git`` integration associates...

Workflow
========

> Use the library, ``rubicon_ml``, to capture model inputs and outputs over time. 

Reviewer: ```suggestion
Use ``rubicon_ml`` to capture model inputs and outputs over time. 
```

> It can be easily integrated into existing Python models or pipelines and supports both

Reviewer: ```suggestion
It easily integrates into existing Python models or pipelines and supports both
```

Keeping the same subject throughout the sentence here
